packages/components/pagination/package.json

```
@@ -37,6 +37,7 @@
     "@commercetools-uikit/utils": "11.2.1",
     "@emotion/react": "^11.1.1",
     "@emotion/styled": "^11.0.0",
+    "formik": "2.2.6",
```

Let's use a version range, to avoid possible version conflicts with formik versions used by custom apps dependencies.

```suggestion
    "formik": "^2.2.6",
```